src/rust/engine/process_execution/src/remote.rs

```diff
@@ -897,6 +897,12 @@ pub fn make_execute_request(
       v => v,
     });
 
+  // Sort the environment variables. REv2 spec requires sorting by name for same reasons that
+  // platform properties are sorted, i.e. consistent hashing.
+  command
```

Comment:
We store the env in a `BTreeMap`, so it should already be sorted?

Reply:
Note that we flatten that BTreeMap to a `Vec` of `bazel_protos::remote_execution::Command_EnvironmentVariable` and then may append `CACHE_KEY_GEN_VERSION_ENV_VAR_NAME` onto the end of the list. Thus, the Vec technically does need to be sorted again.